src/Mysql.php

```diff
@@ -30,11 +30,13 @@ class Mysql
      * @param $database
      * @param int $fetchMode
      * @param string $charset
+     * @param array $pdo_options
      * @param array $options
      *
      * @throws MysqlException
      */
-    public function __construct($host, $user, $password, $database, $fetchMode = \PDO::FETCH_ASSOC, $charset = 'utf8', array $options = [])
+    public function __construct($host, $user, $password, $database, $fetchMode = \PDO::FETCH_ASSOC,
+                                $charset = 'utf8', array $pdo_options = array(), array $options = [])
```

Comment:
Your connection will stay open as long as your script runs or you are closing it yourself with `$mysql->close()`. How does your script look like?